projects/packages/autoloader/src/class-container.php

```diff
@@ -126,16 +130,5 @@ private function initialize_globals() {
 		if ( ! isset( $jetpack_autoloader_activating_plugins_paths ) ) {
 			$jetpack_autoloader_activating_plugins_paths = array();
 		}
-
-		// Since older autoloaders include newer ones, we need to be able to tell the difference between an
-		// inclusion via autoloader and an inclusion via plugin file. This allows the autoloader to
-		// perform special tasks for each kind of inclusion.
-		global $jetpack_autoloader_including_latest;
```

Comment:
Since this global only exists in the `2.7.0` to `2.8.0` versions of the autoloader it has very limited usefulness. I chose to remove it entirely based on the observation that its status can be more accurately inferred from other globals as-needed. I tend to lean towards removing as much cross-autoloader state as possible so that it is easier to reason about how the autoloader works.

Reply:
I think it makes sense to remove it, but will removing it affect backward compatibility with 2.7.0 and 2.8.0? I'm not sure.

Reply:
Since the global was only ever set when loading a _newer_ autoloader we won't experience any problems.